<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperBlockWindow</class>
 <widget class="QDialog" name="SuperBlockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>1048</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="s_r_blocks_count44">
         <property name="text">
          <string>s_r_blocks_count</string>
         </property>
        </widget>
       </item>
       <item row="36" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>s_journal_inum</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="s_frags_per_group">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>s_lastcheck</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="s_wtime">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="38" column="1">
        <widget class="QLabel" name="s_last_orphan">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="29" column="1">
        <widget class="QLabel" name="s_feature_ro_compat">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="s_log_frag_size">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QLabel" name="s_checkinterval">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="25" column="1">
        <widget class="QLabel" name="s_inode_size">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="40" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>s_def_hash_version</string>
         </property>
        </widget>
       </item>
       <item row="39" column="1">
        <widget class="QLabel" name="s_hash_seed">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>s_feature_ro_compat</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>s_blocks_per_group</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>s_first_data_block</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="s_inodes_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="33" column="1">
        <widget class="QLabel" name="s_algo_bitmap">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>s_blocks_count</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>s_mnt_count</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>s_log_frag_size</string>
         </property>
        </widget>
       </item>
       <item row="42" column="1">
        <widget class="QLabel" name="s_first_meta_bg">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="31" column="1">
        <widget class="QLabel" name="s_volume_name">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="41" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>s_default_mount_options</string>
         </property>
        </widget>
       </item>
       <item row="28" column="1">
        <widget class="QLabel" name="s_feature_incompat">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="38" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>s_last_orphan</string>
         </property>
        </widget>
       </item>
       <item row="25" column="0">
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>s_inode_size</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLabel" name="s_lastcheck">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>s_def_resgid</string>
         </property>
        </widget>
       </item>
       <item row="39" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>s_hash_seed</string>
         </property>
        </widget>
       </item>
       <item row="27" column="0">
        <widget class="QLabel" name="label_38">
         <property name="text">
          <string>s_feature_compat</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>s_rev_level</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="s_mnt_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>s_minor_rev_level</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>s_errors</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>s_magic</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="s_free_inodes_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="41" column="1">
        <widget class="QLabel" name="s_default_mount_options">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>s_first_ino</string>
         </property>
        </widget>
       </item>
       <item row="27" column="1">
        <widget class="QLabel" name="s_feature_compat">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="s_errors">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>s_state</string>
         </property>
        </widget>
       </item>
       <item row="35" column="1">
        <widget class="QLabel" name="s_journal_uuid">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="30" column="0">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>s_uuid</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <widget class="QLabel" name="s_def_resgid">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>s_checkinterval</string>
         </property>
        </widget>
       </item>
       <item row="24" column="1">
        <widget class="QLabel" name="s_first_ino">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="33" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>s_algo_bitmap</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QLabel" name="s_rev_level">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="42" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>s_first_meta_bg</string>
         </property>
        </widget>
       </item>
       <item row="40" column="1">
        <widget class="QLabel" name="s_def_hash_version">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>s_mtime</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>s_inodes_count</string>
         </property>
        </widget>
       </item>
       <item row="37" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>s_journal_dev</string>
         </property>
        </widget>
       </item>
       <item row="36" column="1">
        <widget class="QLabel" name="s_journal_inum">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>s_block_group_nr</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="s_free_blocks_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="35" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>s_journal_uuid</string>
         </property>
        </widget>
       </item>
       <item row="32" column="1">
        <widget class="QLabel" name="s_last_mounted">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="31" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>s_volume_name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="s_log_block_size">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>s_free_inodes_count</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>s_log_block_size</string>
         </property>
        </widget>
       </item>
       <item row="37" column="1">
        <widget class="QLabel" name="s_journal_dev">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="34" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>s_prealloc_dir_blocks</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>s_max_mnt_count</string>
         </property>
        </widget>
       </item>
       <item row="26" column="1">
        <widget class="QLabel" name="s_block_group_nr">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QLabel" name="s_creator_os">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="s_magic">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="s_first_data_block">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="s_mtime">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>s_free_blocks_count</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="s_blocks_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="s_max_mnt_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="34" column="1">
        <widget class="QLabel" name="s_prealloc_dir_blocks">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="s_r_blocks_count">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>s_feature_incompat</string>
         </property>
        </widget>
       </item>
       <item row="32" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>s_last_mounted</string>
         </property>
        </widget>
       </item>
       <item row="30" column="1">
        <widget class="QLabel" name="s_uuid">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="s_state">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="s_minor_rev_level">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>s_creator_os</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="s_blocks_per_group">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>s_frags_per_group</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>s_wtime</string>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="label_43">
         <property name="text">
          <string>s_def_resuid</string>
         </property>
        </widget>
       </item>
       <item row="22" column="1">
        <widget class="QLabel" name="s_def_resuid">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
